getting_started.md

```diff
@@ -0,0 +1,138 @@
+
+![](resources/images/AddComponent_010.png)
+
+We will select the "repeated single pixel shape" here and enter the rows and columns as 16. The row and column heights can be left as their default values.
+
+
+When hitting add, you will notice that the 3D view is updated should contain a grid of 16x16 cylinders. We now just need to move it backwards from the sample by applying another transformation of 5 metres along the z-axis.
+
+![](resources/images/NeXusConstructor_012.png)
+
+### Adding a Monitor
+
+We will add a monitor with a mesh shape to the instrument in this example. To do this, add another component.
+When the Add component Dialog is shown, select "NXmonitor" as the component type.
+
+After this, select "Mesh" as the "Shape Type".
+
+##### Loading shape from a CAD file
+Alternatively to cylinder shapes, mesh shapes can be used for components to describe their shape. Currently STL and OFF files are supported for mesh geometry.
+
+To use a mesh file for geometry, select the "Mesh" geometry type. This will show an option to browse for a file. As well as this, units can also be set to change the scale of the geometry.
+To test this, there are OFF and STL files in the `tests/` folder. Let's use the "cube.off" file.   
+
+![](resources/images/AddComponent_011.png)
+
+Again, this would need to be moved away from the sample if reflecting a real scenario. It may also need to be rotated. To add multiple transformations, select "Add rotation" or "Add translation" in the order you want them to be added.
+
+We now have a very basic example of a beamline instrument visualised in 3D as well as the resulting NeXus file. The NeXus Constructor can output to a JSON as well, which is used by the file-writer to aggregate event data and other live-streamed data to the file.
+
+### Opening and saving file-writer commands
+
+The constructor can output to a [file-writer command](https://github.com/ess-dmsc/kafka-to-nexus/blob/master/documentation/commands.md) in JSON format, and can also load from these files.
+To save to a JSON file, open the file menu and click "Save to filewriter JSON". This will bring up a file dialog in order to save the file.
+
+To open a JSON file, open the file menu and click "Open from filewriter JSON". The constructor will then load in all components and their fields from a given file.
+
+#### Sending a command to the file-writer
+
+As well as saving the NeXus structure to file, a run start message can be constructed and sent to an instance of the [file-writer](https://github.com/ess-dmsc/kafka-to-nexus)
+To do this, open the file menu and select "Control file writer". This will bring up a dialog that enables a user to input [Kafka topic](https://kafka.apache.org/documentation/#intro_topics) URIs to look at the status of any active file-writers or to send a command to one.
+
+![](resources/images/nexus-constructor.py_013.png)
+
+### Opening and saving to NeXus File
+
```

Comment:
I suggest replacing the contents of this section with:
"The NeXus Constructor can also directly create a NeXus file. However, note that it will not populate the file with data from Kafka."